avocado/core/streams.py

```diff
@@ -1,5 +1,13 @@
 #: Builtin special keywords to enable set of logging streams
 BUILTIN_STREAMS = {
+    "app": "avocado.app",
+    "test": "avocado.test",
+    "job": "avocado",
```

Comment:
Are we going to collect everything under `avocado` in the job namespace?  I had the impression that `avocado.job` would be mapped to `job` stream.